lib/jekyll/site.rb

```diff
@@ -158,7 +164,6 @@ def render
         end
       end
 
-      payload = site_payload
```

Comment:
Hm. I was doing this to update the contents of the payload. If it still works as expected (they're all pointers I guess) then it should be fine. I'm sorry to be cross – I thought you removed the caching! Thanks, Steve.